data_model/1.3/clusters/ResourceMonitoring.xml

```diff
@@ -73,7 +73,7 @@ Davis, CA 95616, USA
     <feature bit="1" code="WRN" name="Warning" summary="Supports warning indication">
       <optionalConform/>
     </feature>
-    <feature bit="2" code="REP" name="Replacement Product List" summary="Supports specifying the list of replacement products">
+    <feature bit="2" code="REP" name="Replacement" summary="Supports specifying the list of replacement products">
```

Comment:
Oh this is a feature with spaces in the name, need to be fixed. Noted.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/10078